src/patternfly/components/Select/select.scss

```diff
@@ -27,9 +27,6 @@
   --pf-c-select__toggle--m-plain--Color: var(--pf-global--Color--100);
   --pf-c-select__toggle--m-plain--hover--Color: var(--pf-global--Color--100);
 
-  // When there is a typeahead (input box) adjust the top padding of the wrapper in case it wraps to the second line
-  --pf-c-select__toggle-wrapper--m-typeahead--PaddingTop: var(--pf-global--spacer--xs);
```

Comment:
Instead of removing this, can we just set it to `0` and create a follow-up issue for a breaking change release to remove the var?

Reply:
Updated